src/protagonist/DLCS.Model/Assets/AssetDeliveryChannels.cs

```diff
@@ -45,6 +45,12 @@ public static bool HasSingleDeliveryChannel(this Asset asset, string deliveryCha
            asset.ImageDeliveryChannels.Count == 1 && 
            asset.HasDeliveryChannel(deliveryChannel);
     
+    /// <summary>
+    /// Checks if an asset has any delivery channel specified in a list
+    /// </summary>
+    public static bool HasAnyDeliveryChannel(this Asset asset, params string[] deliveryChannels)
+        => deliveryChannels.Any(asset.HasDeliveryChannel);

```

Comment:
This method impl feels like the most convenient way to do this, rather than the most efficient. It won't have a huge effect because `ImageDeliveryChannels` is fairly small but for every `deliveryChannels` passed:

* The `if (asset.ImageDeliveryChannels.IsNullOrEmpty()) return false;` check will happen
* `asset.ImageDeliveryChannels.Any` call will happen meaning multiple iterations.